resources/rulesets.yaml

```diff
@@ -184,3 +184,4 @@ items:
         - 10667615-e067-45b4-a925-563a79ea0949
         - 1b2a0eef-511c-4c07-a36a-d301b15d8207
       checksum: 3668a741fc5025557a192b7ab5712237a718aefac99c19248d62fb88151c0c5c
+root: ./resources
```

Comment:
I am unsure about this change in rulesets.yaml file. Can someone tell if this is correct or not?

Reply:
@pranavgaikwad This doesn't belong here, what is the intention of adding this?

Reply:
I just ran `go run ./cmd/ruleset/` and it added that.